src/OrchardCore.Modules/OrchardCore.Contents/Views/Contents-Button-Actions.SummaryAdmin.cshtml

```diff
@@ -1,38 +1,42 @@
+@using OrchardCore.Contents
 @inject IAuthorizationService AuthorizationService
 @inject IContentManager ContentManager
 
 @{
     ContentItem contentItem = Model.ContentItem;
     var hasPublished = await ContentManager.HasPublishedVersionAsync(contentItem);
     var hasDraft = contentItem.HasDraft();
-    var hasClonePermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.CloneContent, contentItem);
-    var hasDeletePermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.DeleteContent, contentItem);
-    var hasEditPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.EditContent, contentItem);
-    var hasPreviewPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.PreviewContent, contentItem);
-    var hasPublishPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.Permissions.PublishContent, contentItem);
+    var hasDeletePermission = await AuthorizationService.AuthorizeAsync(User, CommonPermissions.DeleteContent, contentItem);
```

Comment:
So you changed everything to use `CommonPermissions` everywhere?

Did you keep the permissions file in `Contents` but not in `Preview`?

Reply:
I removed the one in the Preview module because the other permission in the Contents module or the CommonPermissions.PreviewContent was not used anywhere.

Reply:
Here we need to add the check for permissions on wether a user can or cannot Export as JSON from the deployment module.

Reply:
Else the other way around would be to specify each time `OrchardCore.Contents.CommonPermissions.SomePermission`